### PR TITLE
feature/128/api

### PR DESCRIPTION
**Pull request by theapache64**
PR regarding #128

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+|property_type_strategy|false|String|AutoDeterMineNullableOrNot|Possible values are `NotNullable`, `Nullable`,`AutoDeterMineNullableOrNot`.|
+
+
+**Response Body**
+```json
+{
+  "data" : {
+    "code" : "data class MyJsonClass(\n    val name: String\n)"
+  },
+  "error" : true,
```
Comment (wuseal) [inline api/README.md]: should this error value be false?

File: library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package wu.seal.jsontokotlin.library
+
+fun main(args: Array<String>) {
```
Comment (wuseal) [inline library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt]: What is this used for ?

Comment (theapache64) [reply library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt]: That just a temp file to check if the library integration works or not. If the integration got any issue, the program won't compile.

File: api/README.md (hunk 1 above)
Comment (theapache64) [reply api/README.md]: Yes my mistake. Will fix it.

File: library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt (hunk 2 above)
Comment (wuseal) [reply library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt]: Dose this file  will be compiled int o library?

Comment (theapache64) [reply library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt]: Yes it does.

**Pull request merged**